docs/guide/dev/integration/vue/README.md

```diff
@@ -0,0 +1,15 @@
+---
+category: integration
+order: 40
+url: guide/dev_vue
+menu-title: Vue Integration
+meta-title-short: Vue Integration
+---
+<!--
+Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# CKEditor 4 WYSIWYG Editor Vue Integration
+
+The Vue Integration is currently being developed. Progress of our works can be found in the [CKEditor4-Vue repository](https://github.com/ckeditor/ckeditor4-vue). You can also leave us with feedback in the [Vue Integration Issue page](https://github.com/ckeditor/ckeditor-dev/issues/2861).
```

Comment:
```suggestion
The CKEditor 4 Vue Integration is currently under development. Progress of our work can be tracked in the [CKEditor4-Vue repository](https://github.com/ckeditor/ckeditor4-vue).

We are open for any valuable feedback. Don't hesitate to post any ideas or feature requests in  the [Vue Integration Issue page](https://github.com/ckeditor/ckeditor-dev/issues/2861).
```

Reply:
I wonder if feedback section shouldn't also link to `https://github.com/ckeditor/ckeditor4-vue/issues` instead of `https://github.com/ckeditor/ckeditor-dev/issues/2861`. WDYT?

OTOH, there are no issues there at all, which might seems confusing.

Reply:
To me https://github.com/ckeditor/ckeditor-dev/issues/2861 is for general feedback and questions.
And https://github.com/ckeditor/ckeditor4-vue/issues only for feature request/bug reports.

We don't want the repo to be spammed with question like: 
> When are you going to publish it?
When #2861 is perfect place such questions.

Probably it would be best to have both links. WDYT?

Reply:
> Probably it would be best to have both links. WDYT?

In fact we have both already, the first link in the section links to Vue repo and the second to `ckeditor-dev` general issue. 🤔 Maybe while the integration is still in progress #2861 is better for the feedback as you mentioned.